lib/nexpose/password_policy.rb

```diff
@@ -0,0 +1,69 @@
+module Nexpose
+  require 'json'
+  # Configuration structure for password policies.
+  class Password_policy < APIObject
+    # Policy name
+    attr_accessor :policy_name
+    # Default min length = 6.
+    attr_accessor :min_length
+    # Max length of password can reach upto 32 characters.
+    attr_accessor :max_length
+    # Between 0-4 capital letters can be included in a password
+    attr_accessor :capitals
+    # Between 0-4 digits can be included in a password
+    attr_accessor :digits
+    # Between 0-4 special characters can be included in a password
+    attr_accessor :special_chars
+    # Between 30-120 days can be set for expiry time
+    attr_accessor :expiry_days
+
+    def initialize(name, minlength, maxlength, specialchars, capitals, digits, expirydays)
```

Comment:
While `initialize` could use some defaults, I think built-in policies are entirely out of scope of the gem.